Readme.md

```diff
@@ -58,7 +58,7 @@
         ```
 - Run application.
     ```
-    python manage.py runserver
+   
```

Comment:
Why is manage command for starting server removed from the readme?

Reply:
@PatrickCmd ,Thank you.It was a typo.Let me fix it